client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -2,38 +2,223 @@
+import NotificationsOrigin from './components/notifications-origin';
+
+const fromName =
+	{
+		field: 'woocommerce_email_from_address',
+		title: translate( 'From name' ),
+		subtitle: translate( 'Emails will appear in recipients inpboxses \'from\' this name.' ),
+	};
+const fromAddress =
+	{
+		field: 'woocommerce_email_from_name',
+		title: translate( 'From address' ),
+		subtitle: translate( 'If recipients reply to store emails they will be sent to this address.' ),
+	};
+
+const internalNotifications = [
+	{
+		field: 'email_new_order',
+		title: translate( 'New order' ),
+		subtitle: translate( 'Sent when a new order is received.' ),
+	},
+	{
+		field: 'email_cancelled_order',
+		title: translate( 'Cancelled order' ),
+		subtitle: translate( 'Sent when a new order is marked \'cancelled\'.' ),
+	},
+	{
+		field: 'email_failed_order',
+		title: translate( 'Failed order' ),
+		subtitle: translate( 'Sent when a new order is marked \'failed\'.' ),
+	},
+];
+
+// id enabled
+const customerNotifications = [
+	{
+		field: 'email_customer_on_hold_order',
+		title: translate( 'Order pending payment' ),
+		subtitle: translate( 'Sent when an order is marked \'payment pending\'.' ),
+	},
+	{
+		field: 'email_customer_processing_order',
+		title: translate( 'Processing order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment processing\'.' ),
+	},
+	{
+		field: 'email_customer_completed_order',
+		title: translate( 'Completed order' ),
+		subtitle: translate( 'Sent when an order is marked \'paid in full\'.' ),
+	},
+	{
+		field: 'email_customer_refunded_order',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+	{
+		field: 'email_customer_new_account',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+];
 
 class Settings extends React.Component {
 
+	constructor( props ) {
+		super( props );
+	}
+
+	fetchSettings = ( props ) => {
+		const { siteId, fetchSettings } = props;
+		siteId && fetchSettings( siteId );
+	}
+
 	componentDidMount = () => {
-		const { siteId, fetchEmailSettings: fetch } = this.props;
-		siteId && fetch( siteId );
+		this.fetchSettings( this.props );
 	};
 
 	componentWillReceiveProps = newProps => {
-		if ( newProps.siteId === this.props.siteId ) {
-			return;
+		if ( newProps.siteId !== this.props.siteId ) {
+			this.fetchSettings( newProps );
 		}
+	};
+
+	notificationsToggle = () => {
+		undefined;
+	}
 
-		const { siteId, fetchEmailSettings: fetch } = newProps;
-		siteId && fetch( siteId );
+	recipientsChange = () => {
+		undefined;
 	}
 
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	render() {
+		const { loading, loaded, settings } = this.props;
+		const waiting = ! loaded || loading;
+
 		return (
 			<div className="email-settings__container">
+				<div className="email-settings__origin">
+					<Card className="email-settings__origin-title">
```

Comment:
The title should not be a separate card. This should be using `ExtendedHeader` instead.